File: cryptanil/src/main/java/com/primesoft/cryptanil/utils/StringUtils.java
package com.primesoft.cryptanil.utils;

public class StringUtils {

    public static String replaceCommas(String string) {
        return string.replace(",", ".");
    }

    public static boolean isValidDouble(String s) {
        return !(s.isEmpty() || s.equals("-") || s.equals(".") || s.contains("-.") || s.contains(".-"));
    }

}

File: cryptanil/src/main/java/com/primesoft/cryptanil/utils/StringUtils.kt
package com.primesoft.cryptanil.utils

class StringUtils {
    companion object{
        fun replaceCommas(string: String): String {
            return string.replace(",", ".")
        }

        fun isValidDouble(s: String): Boolean {
            return !(s.isEmpty() || s == "-" || s == "." || s.contains("-.") || s.contains(".-"))
        }
    }
}